src/olympia/devhub/templates/devhub/includes/addons_edit_nav.html

```diff
@@ -29,6 +29,15 @@
         <li {% if url in request.path|urlencode %}class="selected"{% endif %}>
           <a href="{{ url }}">{{ title }}</a></li>
       {% endfor %}
+
+      {% if addon.promoted_subscription and addon.promoted_subscription.stripe_checkout_completed %}
+      <li class="stripe-customer-portal">
+        <form method="POST" action="{{ url('devhub.addons.subscription_customer_portal', addon.slug) }}">
```

Comment:
Why does this need to be a POST, when all the other links are just `<a>` tags?

Reply:
This is what Stripe recommends to initiate the portal creation.